woocommerce/wc-admin
# Reverts work done in #4857 for automated shipping after OBW is completed

Smart defaults were reverted due to issues highlighted in https://github.com/woocommerce/woocommerce-admin/issues/5602. Related reverts https://github.com/woocommerce/woocommerce-admin/pull/5720 and https://github.com/woocommerce/woocommerce-admin/pull/5721 were made but did not address the changes done for automated shipping after OBW completion. This PR would remove automated shipping zone creation.

Original PR: https://github.com/woocommerce/woocommerce-admin/pull/4857
Original task: https://github.com/woocommerce/woocommerce-admin/issues/4595

Fixes #5953

### Detailed test instructions:

- In a fresh WC installation, complete OBW with the country set to US
- Connect to Jetpack and WCS after completing the theme step
- See that shipping task is not set to complete
- Additionally, to confirm that no shipping zone is created, you can go to `WooCommerce > Settings > Shipping` and verify
